manim/utils/tex_file_writing.py

```diff
@@ -79,18 +79,18 @@ def generate_tex_file(expression, environment=None, tex_template=None):
         output = tex_template.get_texcode_for_expression(expression)
 
     tex_dir = config.get_dir("tex_dir")
-    if not os.path.exists(tex_dir):
-        os.makedirs(tex_dir)
+    if not Path(tex_dir).exists():
+        Path(tex_dir).mkdir()
```

Comment:
```suggestion
    if not tex_dir.exists():
        tex_dir.mkdir()
```
`config.get_dir()` already returns a Path.